common/app/templates/headerInlineJS/membershipAccess.scala.js

```diff
@@ -26,23 +27,28 @@
                 window.location.href = membershipAuthUrl;
             }
 
-            ajax({
-                url: membershipUrl + '/user/me',
-                type: 'json',
-                crossOrigin: true,
-                withCredentials: true
-            }).then(function (resp) {
-                // Check the users access matches the content
-                var canViewContent = (requiresPaidTier) ? !!resp.tier && resp.isPaidTier : !!resp.tier;
-                if (canViewContent) {
-                    $('body').removeClass('has-membership-access-requirement');
-                } else {
+            if (id.isUserLoggedIn()) {
+		    ajax({
```

Comment:
indentation is slightly off, and I'd rename `id` to something easier to understand, `id` is too short and generic, maybe `identity`?